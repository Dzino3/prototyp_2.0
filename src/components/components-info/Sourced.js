import React from 'react'

function Sourced(sourcedInfo) {
  return (
    <div>
      
      <h2>{sourcedInfo.sourcedInfo.ORDER_NUMBER}</h2>
        <div className='orderInfo'>
            <div className='infoSourcedLabel'>
                <h3 className='orderH3'>Sourced</h3>
                <label className='reqInfo'>Base Code Description:</label>
                <label className='reqInfo'>Factory Delivery:</label>
                <label className='reqInfo'>Original Factory Delivery:</label>
                <label className='reqInfo'>Requested Delivery:</label>
                <label className='reqInfo'>FDD-RDD:</label>
                <label className='reqInfo'>Sold:</label>
                <label className='reqInfo'>Order entry date:</label>
                <label className='reqInfo'>Order source date:</label>
                <label className='reqInfo'>Machine status:</label>
                <label className='reqInfo'>Order staus:</label>
                <label className='reqInfo'>Req. ship date:</label>
                <label className='reqInfo'>Pres date:</label>
                <label className='reqInfo'>Qty.:</label>
                <label className='reqInfo'>CC:</label>
            </div>
            <div className='infoSourced'>
                <h3 className='orderH3i'>Info</h3>
                <input type='text' name='info' id='Base Code Descriptioninfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.BASE_CODE_DESCR} disabled></input>
                <input type='text' name='info' id='FDDinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.FDD} disabled></input>
                <input type='text' name='info' id='OFDDinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.OFDD} disabled></input>
                <input type='text' name='info' id='RDDinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.RDD} disabled></input>
                <input type='text' name='info' id='FDD-RDDinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.FDD_MINUS_RDD} disabled></input>
                <input type='text' name='info' id='Soldinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.SOLD} disabled></input>
                <input type='text' name='info' id='Order entry dateinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.ORDER_ENTRY_DATE} disabled></input>
                <input type='text' name='info' id='Order source dateinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.ORDER_SOURCE_DATE} disabled></input>
                <input type='text' name='info' id='Machine statusinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.MACHINE_STATUS} disabled></input>
                <input type='text' name='info' id='Order stausinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.ORDER_STATUS} disabled></input>
                <input type='text' name='info' id='Req. ship dateinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.REQ_SHIP_DATE} disabled></input>
                <input type='text' name='info' id='Pres dateinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.PRES_DATE} disabled></input>
                <input type='text' name='info' id='Qtyinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.QTY} disabled></input>
                <input type='text' name='info' id='CCinfo' className='reqInfoOut' placeholder={sourcedInfo.sourcedInfo.CC} disabled></input>
            </div>
        </div>
    </div>
  )
}

export default Sourced
