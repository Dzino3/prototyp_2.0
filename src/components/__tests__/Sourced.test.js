import React from "react";
import renderer from 'react-test-renderer';
import Sourced from '../components-info/Sourced';

describe(`Rendering Sourced`, () => {

    test('should render Sourced with sourcedInfo ', function () {
        let sourcedInfoX = {
            Sourced
        }

        let tree = renderer.create(<Sourced sourcedInfo={sourcedInfoX}></Sourced>);

        expect(tree.toJSON()).toMatchSnapshot();
    });

    test('should render Sourced with empty sourcedInfo ', function () {
        let sourcedInfoX = {}

        let tree = renderer.create(<Sourced sourcedInfo={sourcedInfoX}></Sourced>);

        expect(tree.toJSON()).toMatchSnapshot();
    });
})