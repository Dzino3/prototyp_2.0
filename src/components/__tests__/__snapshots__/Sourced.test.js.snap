// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Sourced should render Sourced with empty sourcedInfo  1`] = `
<div>
  <div
    className="orderInfo"
  >
    <div
      className="infoSourcedLabel"
    >
      <h3
        className="orderH3"
      >
        Sourced
      </h3>
      <label
        className="reqInfo"
      >
        ACC:
      </label>
      <label
        className="reqInfo"
      >
        Base Code Description:
      </label>
      <label
        className="reqInfo"
      >
        Factory Delivery:
      </label>
      <label
        className="reqInfo"
      >
        Original Factory Delivery:
      </label>
      <label
        className="reqInfo"
      >
        Requested Delivery:
      </label>
      <label
        className="reqInfo"
      >
        FDD-RDD:
      </label>
      <label
        className="reqInfo"
      >
        Sold:
      </label>
      <label
        className="reqInfo"
      >
        Order entry date:
      </label>
      <label
        className="reqInfo"
      >
        Order source date:
      </label>
      <label
        className="reqInfo"
      >
        Machine status:
      </label>
      <label
        className="reqInfo"
      >
        Order staus:
      </label>
      <label
        className="reqInfo"
      >
        Req. ship date:
      </label>
      <label
        className="reqInfo"
      >
        Pres date:
      </label>
      <label
        className="reqInfo"
      >
        Qty.:
      </label>
    </div>
    <div
      className="infoSourced"
    >
      <h3
        className="orderH3i"
      >
        Info
      </h3>
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Base Code Descriptioninfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="FDDinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="OFDDinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="RDDinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="FDD-RDDinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Soldinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Order entry dateinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Order source dateinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Machine statusinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Order stausinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Req. ship dateinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Pres dateinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Qtyinfo"
        name="info"
        type="text"
      />
    </div>
  </div>
</div>
`;

exports[`Rendering Sourced should render Sourced with sourcedInfo  1`] = `
<div>
  <div
    className="orderInfo"
  >
    <div
      className="infoSourcedLabel"
    >
      <h3
        className="orderH3"
      >
        Sourced
      </h3>
      <label
        className="reqInfo"
      >
        ACC:
      </label>
      <label
        className="reqInfo"
      >
        Base Code Description:
      </label>
      <label
        className="reqInfo"
      >
        Factory Delivery:
      </label>
      <label
        className="reqInfo"
      >
        Original Factory Delivery:
      </label>
      <label
        className="reqInfo"
      >
        Requested Delivery:
      </label>
      <label
        className="reqInfo"
      >
        FDD-RDD:
      </label>
      <label
        className="reqInfo"
      >
        Sold:
      </label>
      <label
        className="reqInfo"
      >
        Order entry date:
      </label>
      <label
        className="reqInfo"
      >
        Order source date:
      </label>
      <label
        className="reqInfo"
      >
        Machine status:
      </label>
      <label
        className="reqInfo"
      >
        Order staus:
      </label>
      <label
        className="reqInfo"
      >
        Req. ship date:
      </label>
      <label
        className="reqInfo"
      >
        Pres date:
      </label>
      <label
        className="reqInfo"
      >
        Qty.:
      </label>
    </div>
    <div
      className="infoSourced"
    >
      <h3
        className="orderH3i"
      >
        Info
      </h3>
      <input
        className="reqInfoOut"
        disabled={true}
        id="ACCinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Base Code Descriptioninfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="FDDinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="OFDDinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="RDDinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="FDD-RDDinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Soldinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Order entry dateinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Order source dateinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Machine statusinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Order stausinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Req. ship dateinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Pres dateinfo"
        name="info"
        type="text"
      />
      <input
        className="reqInfoOut"
        disabled={true}
        id="Qtyinfo"
        name="info"
        type="text"
      />
    </div>
  </div>
</div>
`;
